papers/bib.py

```diff
@@ -1545,6 +1549,9 @@ def gitcmd(o):
     if hasattr(o,'dry_run'):
         papers.config.DRYRUN = o.dry_run
 
+    if o.version:
+        print(papers.__version__)
```

Comment:
Here I insist that the program should exit. All programs I'm used to have that behaviour -- call me a conformist.